### ISSUE TITLE
dependencyOverrides ignored

### ISSUE THREAD

**Issue by nadavwr** [rtimush/sbt-updates#95]
I specify version ranges in my dependencies, then use `sbt-lock` to generate `lock.sbt` files. Those define a `dependencyOverrides` setting with concrete dependency versions, that override version  ranges. **tl;dr** `libraryDependencies` with version ranges and `dependencyOverrides` with specific versions.

When invoking `dependencyUpdates`, I see dependency updates based on version ranges instead of version overrides.

Example:

```
// plugins.sbt
addSbtPlugin("com.timushev.sbt" % "sbt-updates" % "0.3.+")

// build.sbt
libraryDependencies += "com.lihaoyi" %% "pprint" % "0.5.+"
dependencyOverrides += "com.lihaoyi" % "pprint_2.12" % "0.5.3"
```

And then:

```
$ sbt 'show update'
...
[info] 	com.lihaoyi:pprint_2.12
[info] 		- 0.5.3
...

$ sbt dependencyUpdates
...
[info]   com.lihaoyi:pprint           : 0.5.+            -> 0.5.3
...
```

**Comment by rtimush** [comment]
Indeed it makes sense to take `dependencyOverrides` into account. If you or someone else could make a PR that would be really helpful. Otherwise, I will hopefully take a look myself.

**Comment by seveneves** [comment]
Hey @rtimush 

Could you please review the PR and consider merging it to master and releasing it with next release?

**Comment by rtimush** [comment]
Hey @seveneves. Thanks a lot! I will take a look.

**Comment by nadavwr** [comment]
@rtimush I gave 0.3.5-17 a try and it works for me. Do you plan on releasing it soon?

**Comment by rtimush** [comment]
@nadavwr Thank you for checking. I faced some issues with Travis builds, but I will release a new version soon.

**Comment by rtimush** [comment]
Fixed in 0.4.0
